app/helpers/api/user.ts

```diff
@@ -7,13 +7,14 @@ export const getNeededAtMentionedUsernames = (usernames: Set<string>, posts: Pos
     const usernamesToLoad = new Set<string>();
 
     const pattern = /\B@(([a-z0-9_.-]*[a-z0-9_])[.-]*)/gi;
+    const spcialMentions = new Set<string>(General.SPECIAL_MENTIONS);
```

Comment:
Seems like this is only used here, and I see no reason not to make it a set directly on the constants file.